### PR TITLE
[libbs2b] Add support for clangarm64

### PR DESCRIPTION
**Pull request by stephenjust**
Needed to run autoreconf for aarch64 to build.

### CODE REVIEW

File: mingw-w64-libbs2b/PKGBUILD [hunk 1]
```diff
@@ -17,6 +17,10 @@ sha256sums=('6aaafd81aae3898ee40148dd1349aab348db9bfae9767d0e66e0b07ddd4b2528')
 
 prepare() {
   cd "$srcdir"/${_realname}-${pkgver}
+
+  sed -i 's/dist-lzma/dist-xz/g' configure.ac
```
Comment (Biswa96) [inline mingw-w64-libbs2b/PKGBUILD:21]: Would you like to explain why this change is required?

Comment (stephenjust) [reply mingw-w64-libbs2b/PKGBUILD:21]: Latest autoconf deprecated dist-lzma. I just used the same solution from upstream Arch pkgbuild.

Comment (Biswa96) [reply mingw-w64-libbs2b/PKGBUILD:21]: ok, seems good.
